tasks/uninstall.yml

```diff
@@ -5,6 +5,7 @@
   with_items:
     - {service: "{{ funkwhale_frontend_identifier }}.service", path: "{{ devture_systemd_docker_base_systemd_path }}/{{ funkwhale_frontend_identifier }}.service"}
     - {service: "{{ funkwhale_api_identifier }}.service", path: "{{ devture_systemd_docker_base_systemd_path }}/{{ funkwhale_api_identifier }}.service"}
+    - {service: "{{ funkwhale_watch_imports_identifier }}.service", path: "{{ devture_systemd_docker_base_systemd_path }}/{{ funkwhale_watch_imports_identifier }}.service", when: funkwhale_watch_imports_enabled}
```

Comment:
It probably doesn't hurt to remove the `when` and to always ensure uninstallation removes all files.

Reply:
Actually, `funkwhale-celery-worker` and `funkwhale-celery-beat` should be included as well I suppose.

Reply:
Good point! I believe you're right